.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,16 @@
 [Describe your change]
+Got some code for us? Awesome 🎊!
 
+Please include a description of your change & check your PR against this list, thanks!
 
+- [ ] If it resolves a ticket, please add `Resolves #[Issue number]`
+- [ ] Commit messages are meaningful
+- [ ] The code is formatted using `cargo fmt`
+- [ ] The project compiles: `cargo make build`
 
-
-Resolves #[Issue number]
+- [ ] You agree to the [Contributor License Agreement](../CONTRIBUTING.md)
```

Comment:
❌ This does not point to the CLA

Reply:
CLA is in `CONTRIBUTING.md`... I wanted to link to that section directly (e.g. (`../CONTRIBUTING.md#CLA`), but this does not work :(